frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -144,6 +146,35 @@ function ReaderCoptListener:updatePageInfoOverride(pageno)
     if self.reading_percent == 1 then
         page_info = page_info .. percentage_pre .. percentage_fmt:format(percentage*100) .. percentage_post
     end
+
+    if self.battery == 1 and self.battery_percent == 1 then -- append battery percentage
+        local batt_pre = "["
+        local batt_post = "]"
+        local batt_val = ""
+        if Device:hasBattery() then
+            local powerd = Device:getPowerDevice()
+            local batt_lvl = powerd:getCapacity()
+
+            if Device:hasAuxBattery() and powerd:isAuxBatteryConnected() then
+                local aux_batt_lvl = powerd:getAuxCapacity()
+                batt_pre = "+"
+                batt_post = "+"
+                -- Sum both batteries for the actual text
+                batt_lvl = batt_lvl + aux_batt_lvl
+            else
+                batt_pre = "+"
+                batt_post = "+"
```

Comment:
The doubled `+` feel a bit excessive:
![image](https://github.com/koreader/koreader/assets/24273478/3a405ed3-4829-4eae-9c46-837e4952cc35)
The low weight of the `+` glyph makes it feel like a hole on the right and a bit misaglined. If it ends on the right with the `%`, it looks better, like proper justficiation :) instead of a hanging `+`.